### DISCUSSION TITLE
Possible to use Vue 2.7?

### DISCUSSION THREAD

**Discussion by djkunkel** [vueform/vueform#85] [Questions] [answered]
I see there are some tests in the repo for Vue 2.7, but when I try to use the project with Vue 2.7, I get console errors saying there's an undefined property 'render', 'template', and 'app' which makes me think its fully dependent on 3.

Would it be possible to get a minimum example for Vueform working with 2.7 if its even possible anymore? I'm using Vite with '@vitejs/plugin-vue2'.

Thank you!

**Comment by adamberecz** [comment]
I can see we've introduced this bug for Vue 2 when doing and update for tree-shaking in one of the latest patches. Will get fixed soon!

**Comment by adamberecz** [comment, marked as answer]
Please upgrade to `1.6.4` @djkunkel - it should work in Vue 2 too.

**Reply by djkunkel** [reply]
This works! Thank you for the fast response.

**Comment by JWPersh** [comment]
@djkunkel How did you include VueForm in your Vue 2 project? I've tried the below

```
import Vueform from '@vueform/vueform'
import vueformConfig from '../vueform.config'
Vue.use('Vueform', Vueform)
```
But when referencing the VueFrom component from another Vue component I get an error in the browser console.

```
<Vueform>
      <TextElement name="hello_world" label="Hello" placeholder="World" />
</Vueform>
```
Gives error:
[Vue warn]: Failed to mount component: template or render function not defined.

**Reply by djkunkel** [reply]
I wasn't able to make it work if the root App.vue used <script setup>, but I was able to make it work using the setup() function.

Example:
https://stackblitz.com/edit/vitejs-vite-smqqxt?file=main.js

**Reply by JWPersh** [reply]
Awesome. Thanks!
